spec/modules-spec.js

```diff
@@ -47,6 +47,20 @@ describe('third-party module', function () {
       })
     })
   })
+
+  describe('global variables', function () {
+    describe('process', function () {
+      it('can be declared in a module', function () {
+        require(path.join(fixtures, 'module', 'declare-process.js'))
```

Comment:
For requires for simple relative paths just a string is fine, usually `path.join` is used when the path is more complicated.